sparkbox/designsystemssurvey
# 2022/base styles

This setups the base  styles including colors and typography for the 2022 design.

To test:

1. Run `npm install`
2. Run `npm start`
3. Open [http://localhost:8000/2022](http://localhost:8000/2022) in your browser
4. Review heading styles and body styles against the [Figma styles](https://www.figma.com/file/VcIuWHbkjExLZ8SNQZipAo/2022-DS-Survey?node-id=2207%3A1035)

Check the following styles:
- [x] **Heading Styles**
   - [x] Section Header
   - [x] Header 02
   - [x] Header 03
   - [x] Header 04
- [x] **Body Styles**
   - [x] Body Default
   - [x] Body Default Link
   - [x] Body Large
   - [x] Body Large Regular
   - [x] Body X Large
